.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "root": true,
-  "extends": ["airbnb-base", "prettier"],
+"extends": ["airbnb-base", "prettier"],
```

Comment:
What's happening here?

Reply:
Nothing, its the same as before

Reply:
Then why did you format it? Revert to what it was, since no change intended